omod/src/main/java/org/openmrs/module/registrationapp/web/controller/patientImageController.java

```diff
@@ -0,0 +1,60 @@
+package org.openmrs.module.registrationapp.web.controller;
+
+import java.io.IOException;
+import java.util.Base64;
+import java.util.Iterator;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.emrapi.person.image.EmrPersonImageService;
+import org.openmrs.module.emrapi.person.image.PersonImage;
+import org.openmrs.ui.framework.SimpleObject;
+import org.openmrs.ui.framework.fragment.action.FailureResult;
+import org.openmrs.ui.framework.fragment.action.FragmentActionResult;
+import org.openmrs.ui.framework.fragment.action.ObjectResult;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.MultipartHttpServletRequest;
+
+@Controller
+public class patientImageController {
```

Comment:
Should be capitalized, ie PatientImageController

Reply:
@mogoodrich this is @reagan-meant not @Ruhanga.......Seems our names are similar.
Will fix the name shortly.
Let me look into the option of opt-in too.

Reply:
Whoops, sorry @reagan-meant ... your names aren't that similar, my mistake! I was reviewing a bunch of PRs all at once include a few by @Ruhanga and used autocomplete without checking fully!